extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -70,11 +72,22 @@ private function getNavigation() {
 
 		$navigationNodes = ( new GlobalNavigationHelper() )->getMenuNodes();
 
+		// Add link to explore wikia only for EN language
+		if ( $wgLang->getCode() === WikiaLogoHelper::FANDOM_LANG ) {
+			$navigationNodes[ 'exploreDropdown' ][] = [
+				'text' => wfMessage( 'global-navigation-explore-wikia-mercury-link-label' )->plain(),
+				'textEscaped' => wfMessage( 'global-navigation-explore-wikia-mercury-link-label' )->escaped(),
+				'href' => wfMessage( 'global-navigation-explore-wikia-link' )->plain(),
+				'trackingLabel' => 'explore-wikia'
```

Comment:
I know it's just a revert, but don't we need to update stuff like this? This should be explore-wikis.